backend/FwLite/LcmCrdt/CrdtMiniLcmApi.cs

```diff
@@ -496,6 +509,15 @@ public async Task<Sense> UpdateSense(Guid entryId, Sense before, Sense after)
         return await GetSense(entryId, after.Id) ?? throw new NullReferenceException("unable to find sense with id " + after.Id);
     }
 
+    public async Task<Sense> MoveSense(Guid entryId, Sense sense, BetweenPosition between)
+    {
+        var order = await dataModel.PickOrder<Sense>(between);
+        sense.Order = order;
```

Comment:
Yeah, I don't think this should be here. 👍
Primarily, because Order is an internal property. So callers shouldn't care about the order their sense is assigned.